docs-release-notes.md

```diff
@@ -1,3 +1,17 @@
+# May 24, 2024
+
+## Device Sync
+
+- Configure Sync/Enable Atlas Device Sync: For the Admin API procedure, add a step for enabling Development Mode and clarify procedure steps for enabling sync.
```

Comment:
This probably should be capitalized since we're referring to the product name here.
```suggestion
- Configure Sync/Enable Atlas Device Sync: For the Admin API procedure, add a step for enabling Development Mode and clarify procedure steps for enabling Sync.
```